src/main/java/com/sda/microblogging/exception/InvalidEmailOrPasswordException.java

```diff
@@ -0,0 +1,13 @@
+package com.sda.microblogging.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "User email or password were not found")
+public class InvalidEmailOrPasswordException extends RuntimeException {
+
+    @Override
+    public String getMessage() {
+        return "User email or password were not found";
```

Comment:
Can you the message to "Invalid username or password" too?

Reply:
@bkrmadtya , "Invalid email or password"?

Reply:
Yeah the message, instead of User email or password were not found, the returned message should the same as the name of the exception "Invalid Email or Password".

Reply:
Fixed